Algolia.Search.Test/Algolia.Search.Test.csproj

```diff
@@ -1,14 +1,14 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
 
   <PropertyGroup>
-    <TargetFramework>netcoreapp2.0</TargetFramework>
+    <TargetFramework>netcoreapp2.1</TargetFramework>
```

Comment:
Not needed, but I guess it's better to target the latest framework (for our CI)
It will be better when we will have Azure Pipeline to perform tests for all frameworks.